dbhydra/src/tables.py

```diff
@@ -749,9 +748,8 @@ def create(self, foreign_keys=None):
         print(query)
         try:
             self.db1.execute(query)
-        except Exception as e:
-            print("Table " + self.name + " already exists:", e)
-            print("Check the specification of table columns and their types")
+        except pymysql.OperationalError as e:
+            raise Exception(e.args[1]) from e
```

Comment:
I can't approve this, 
- it was passing these errors now it would crash (happens quite often, not sure what are all the consequences at many places even outside forloop)
- pymysql library should be used as minimally as possible - i.e. I prefer having general exceptions even in a specific dialect of tables/dbs